lms/static/scripts/tsconfig.json

```diff
@@ -9,7 +9,8 @@
     "noEmit": true,
     "strict": true,
     "target": "ES2020",
-    "useUnknownInCatchVariables": false
+    "useUnknownInCatchVariables": false,
+    "esModuleInterop": true
```

Comment:
`esModuleInterop` is necessary for TS to understand the import of a default export: `import debounce from 'loadash/debounce'`